### ISSUE TITLE
Add check to see if WooCommerce is active before initializing anything.

### ISSUE THREAD

**Issue by claudiulodro** [woocommerce/woocommerce-gutenberg-products-block#100]
See https://github.com/WordPress/gutenberg/issues/7423#issuecomment-399046305 for the error that is displayed when the plugin is active without WooCommerce active.

**Comment by claudiosanches** [comment]
Seems like we already only this plugin only in `woocommerce_loaded`: https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/woocommerce-gutenberg-products-block.php#L22-L30

**Comment by claudiulodro** [comment]
[This one calls wc_get_theme_support](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/woocommerce-gutenberg-products-block.php#L79) and loads even if WC isn't loaded. It just needs a small change. 

This issue won't be a problem once this is merged into core. :P

**Comment by claudiulodro** [comment]
See https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/341bb00f6c6f5751594fc47d4c7049284a960470
